### PR TITLE
General cleanup of codebase

### PR DESCRIPTION
**Pull request by ghost**
Fixes #162 

Please update the README so that I can understand how to run the application (FE and BE) properly and test.

### CODE REVIEW

**Comment by flysaiah** [comment]
New README exists in dev branch. Am I right in thinking that 100% of this PR is reformatting due to linting?

**Comment by ghost** [comment]
Reformatted code and also added prettier settings for vscode for further formatting.

**Comment by flysaiah** [comment]
Gotcha. Will finish reviewing this tomorrow and will probably merge unless there are any issues

**Comment by ghost** [comment]
Thanks. I will also update once I test it. 😀

File: src/app/group/group.component.html [hunk 1]
```diff
@@ -19,11 +19,19 @@ <h3 class="group-name">{{ currentGroup['name'] }}</h3>
...
                 <h2 class="group-member-name">{{ groupMember['username'] }}</h2>
-                <p class="group-member-bestgirl" title="Best girl: {{ groupMember['bestgirl'] }}">Best girl: {{ groupMember['bestgirl'] }}</p>
-                <p class="group-member-current-show" title="Watching: {{ groupMember['currentlyWatching'] }}">Watching: {{ groupMember['currentlyWatching'] }}</p>
-                <button class="group-member-button" *ngIf="groupMember['username'] != currentUser" mat-raised-button color="accent" (click)="importAnime(groupMember['username'], groupMember['id'])">Import Anime</button>
-                <button class="group-member-button" *ngIf="groupMember['username'] != currentUser" mat-raised-button color="primary" routerLink="/timeline" [queryParams]="{user: groupMember['username']}">Timeline</button>
-                <button class="group-member-button" *ngIf="groupMember['username'] == currentUser" mat-raised-button color="warn" (click)="leaveGroup()">Leave Group</button>
+                <p class="group-member-bestgirl" title="Best girl: {{ groupMember['bestgirl'] }}">Best girl: {{ groupMember['bestgirl']
+                  }}
```
Comment (flysaiah) [inline src/app/group/group.component.html:23]: I don't like this pattern of splitting up interpolated variables to 2 lines just so we can have the `}}` on a separate line--can this be changed?

File: src/app/group/group.component.html [hunk 2]
```diff
@@ -19,11 +19,19 @@ <h3 class="group-name">{{ currentGroup['name'] }}</h3>
...
+                </p>
+                <p class="group-member-current-show" title="Watching: {{ groupMember['currentlyWatching'] }}">Watching: {{
```
Comment (flysaiah) [inline src/app/group/group.component.html:25]: Same thing here -- I think keeping the Watching: {{ <var> }} is more readable than splitting it into 2 lines

File: src/app/group/group.component.html [hunk 3]
```diff
@@ -19,11 +19,19 @@ <h3 class="group-name">{{ currentGroup['name'] }}</h3>
...
+                  groupMember['currentlyWatching'] }}</p>
+                <button class="group-member-button" *ngIf="groupMember['username'] != currentUser" mat-raised-button color="accent" (click)="importAnime(groupMember['username'], groupMember['id'])">Import
```
Comment (flysaiah) [inline src/app/group/group.component.html:27]: Also dislike splitting button text into 2 lines

**Comment by flysaiah** [comment]
Sorry for the late review, but I'm looking it over now and so far I'm okay with most of the reformatting but I'm not a fan of some of the HTML formatting, particularly when interpolated vars / button text / etc get split into multiple lines; I've commented on a few examples. I'm not sure how this was formatted (I'm assuming using some VSCode prettifier?) so I'm unsure how to change those

**Comment by ghost** [comment]
Thanks for reviewing it. Yes you are right, I have used VSCode prettier plugin. As fat as formatting html files, we can use `//prettier-ignore` above the lines that you are not a fan of and prettier will ignore that lines.

**Comment by flysaiah** [comment]
Is there any way to write a pattern that could match those instances? There are quite a lot of times where this occurs and I don't really want to have to go through every HTML file and add an extra line every time it happens

**Comment by ghost** [comment]
One option is to set the `prettier.printWidth` in .vscode/settings to a larger number. This will prevent these cases.

**Comment by flysaiah** [comment]
Let's try that for now

**Comment by flysaiah** [comment]
Closed for inactivity

**Pull request closed without merging**